Makefile

```diff
@@ -46,13 +46,13 @@ container-build:
 
 container-install: 
 	sudo docker build . -f Dockerfile.build -t runnc-build
-	sudo docker run --rm -v /opt/runnc/lib/:/opt/runnc/lib/ -v /usr/local/bin:/usr/local/bin -v ${PWD}:/go/src/github.com/nabla-containers/runnc -w /go/src/github.com/nabla-containers/runnc runnc-build make install
+	sudo docker run --rm -v /opt/runnc/:/opt/runnc/ -v /usr/local/bin:/usr/local/bin -v ${PWD}:/go/src/github.com/nabla-containers/runnc -w /go/src/github.com/nabla-containers/runnc runnc-build make install
 
 .PHONY: godep
 godep: 
 	dep ensure
 
-build/runnc: godep runnc.go
+build/runnc: godep *.go
```

Comment:
nit: better to list all the `.go` files

Reply:
Read somewhere that with the "new" github, i could propose the change here so you can just click on "accept the change" (can't find it though).